packages/rest/src/__tests__/unit/controller-mixin/controller.mixin.unit.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '@loopback/core';
+import {Client, createClientForHandler, expect} from '@loopback/testlab';
+import {HttpServerLike, RestComponent, RestServer} from '../../..';
+import {
+  groceryNote,
+  NoteController,
+} from '../../fixtures/controllers/note.controller';
+import {Note} from '../../fixtures/models/note.model';
+
+describe('add method to controller via mixin', () => {
+  let controller: NoteController;
+
+  before(async () => {
+    controller = new NoteController();
+  });
+
+  it(`non-mixin method 'create' exists`, () => {
+    expect(controller.create).to.be.a.Function();
+  });
+
+  it(`mixin method 'findByTitle' exists`, async () => {
+    expect(controller.findByTitle).to.be.a.Function();
```

Comment:
Similarly here, this check is redundant and produces less helpful failure message than the next line calling `controller.findByTitle`. Please remove.